src/coll/options.rs

```diff
@@ -574,6 +574,13 @@ pub struct EstimatedDocumentCountOptions {
     /// The level of the read concern.
     #[serde(skip_serializing)]
     pub read_concern: Option<ReadConcern>,
+
+    /// Tags the query with an arbitrary BSON object to help trace the operation through the
+    /// database profiler, currentOp and logs.
+    ///
+    /// This option is only available on server versions 4.4+.
+    #[serde(skip_serializing)]
```

Comment:
We'll want to omit `#[serde(skip_serializing)]` here. We derive `Serialize` on these options structs so that we can serialize the options specified by the user into bson to send the operation to the server. A few of the other options in this struct have `#[serde(skip_serializing)]` because we serialize/use those options manually, but this one can be serialized by the default implementation.